ui.frontend/src/view/FormFieldBase.js

```diff
@@ -222,10 +223,27 @@ class FormFieldBase extends FormField {
            if (this.getErrorDiv() && this.getErrorDiv().innerHTML) {
             appendDescription('errormessage', this.getId());
           }
+
             widgetElement.setAttribute('aria-describedby', ariaDescribedby);
         }
     }
 
+    #syncAriaLabel() {
+        let widgetElement = typeof this.getWidget === 'function' ? this.getWidget() : null;
+        let widgetElements = typeof this.getWidgets === 'function' ? this.getWidgets() : null;
+        widgetElement = widgetElements || widgetElement;
+
+        if (widgetElement && this.getModel() && this.getModel().screenReaderText) {
+            // Create a temporary DOM element to strip HTML tags
+            const tempElement = document.createElement('div');
```

Comment:
Let;s not use DOM API's for this, we already have DOMPurify for this

```
#syncAriaLabel() {
    const widgetElement = this.getWidget?.() || this.getWidgets?.();
    const model = this.getModel?.();

    if (widgetElement && model?.screenReaderText) {
        // Use DOMPurify to sanitize and strip HTML tags
        const screenReaderText = DOMPurify.sanitize(model.screenReaderText, { RETURN_TRUSTED_TYPE: true });
        widgetElement.setAttribute('aria-label', screenReaderText);
    }
}
```

Reply:
Implemented!